src/main/data/contentmodifiers.json

```
@@ -224,5 +224,23 @@
     "dcncCode": "EC",
     "dcncSortOrder": 13,
     "description": "Eclaircolor graded image."
+  },
+  {
+    "cplMetadata": {
+      "attribute": "scope",
+      "attributeValue": "http://www.dcimovies.com/schemas/2018/HDR-Metadata",
+      "definingDocs": [
+        {
+          "name": "DCI High Dynamic Range D-Cinema Addendum, Version 1.0",
+          "url": "https://www.dcimovies.com/specification/DCI-HDR-Addendum-v100_2022-0330.pdf"
```

Let's use the latest HTML version which is under docs.dcimovies.com:

```suggestion
          "url": "https://docs.dcimovies.com/hdr-addendum/f18141e7bf5acfd5ecb90da80261fec3e762bcf7/"
```